Grabber.pro

```diff
@@ -33,8 +33,7 @@ unix:!macx{
 	desktop.files += release/Grabber.desktop
 
 	icon.path = $$PREFIX/share/pixmaps
-	icon.extra = cp resources/icon.png resources/Grabber.png
-	icon.files += resources/Grabber.png
+	icon.extra = install -D -m644 resources/icon.png $(INSTALL_ROOT)/$$PREFIX/share/pixmaps/Grabber.png

```

Comment:
Also qmake automatically uses INSTALL_ROOT on every install command in the Makefile if you're worried about that.